content/en/blog/_posts/2021-05-16-SIG-Usability-Spotlight.md

```diff
@@ -0,0 +1,64 @@
+
+This covers points like user experience and accessibility. The objectives of the SIG are to guarantee that the Kubernetes project is maximally usable by people of a wide range of foundations and capacities, such as incorporating internationalization and ensuring the openness of documentation.
+
+### Q. Why should new and existing contributors consider joining SIG Usability?
+
+A. There are plenty of territories where new contributors can begin. User research projects where people can help understand the usability of the end-user experiences, including error messages, end-to-end tasks, etc.
+
+Accessibility guidelines for Kubernetes community artifacts, examples include internationalization of documentations, color choices for people with color blindness, ensuring compatibility with screen reader technology, user interface design for core components with user interfaces, and more.
+
+### Q. What do you do to help new contributors get started?
+
+A. New contributors can get started by shadowing one of the user interviews, going through user interview transcripts, analyzing them, and designing surveys.
+
+SIG Usability is also open to new project ideas. If you have an idea, we’ll do what we can to support it. There are regular SIG Meetings where people can ask their questions live. These meetings are also recorded for those who may not be able to attend. As always, you can reach out to us on Slack as well.
+
+### Q. What does the survey include?
+
+A. In simple terms, the survey gathers information about how people use Kubernetes, such as trends in learning to deploy a new system, error messages they receive, and workflows.
+
+One of our goals is to standardise the responses accordingly. The ultimate goal is to analyze survey responses for important user stories whose needs aren't being met.
+
+### Q. Are there any particular skills you’d like to recruit for? What skills are contributors to SIG Usability likely to learn?
+
+A. Although contributing to SIG Usability does not have any pre-requisites as such, but experience with user research, qualitative research, prior experience with how to conduct an interview would be a great plus point. Quantitative research, like survey design and screening, is also helpful and something that we expect contributors to learn.
+
+### Q. What are you getting positive feedback on, and what’s coming up next for SIG Usability?
+
+A. We have had new members joining and coming to monthly meetings regularly and showing interests in becoming a contributor and helping the community. We have also had a lot of people reach out to us via Slack showcasing their interest in the sig. 
+
+Currently, we are focused on finishing [this study](https://www.youtube.com/watch?v=Byn0N_ZstE0), which is our project for this year. We are always happy to have new contributors join us.
```

Comment:
Is this reasonable given the answers aren't a transcript?
```suggestion
Currently, we are focused on finishing the study mentioned in our [talk](https://www.youtube.com/watch?v=Byn0N_ZstE0), also our project for this year. We are always happy to have new contributors join us.
```

Reply:
I think the first part of your suggestion is good; but, I think changing "which is" to "also" changes the meaning a touch, suggesting there's another project that's being worked on in addition to the one being referred to in the talk.